lib/langchain/chat.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Langchain
+  class Chat
```

Comment:
What are your thought of calling this class something like `ChatSession`? I don't have any strong feeling one way or another here.